README.md

```diff
@@ -134,17 +134,20 @@ Response
 
 ## PATCH
 
-An array of order item data. The minimum data in order item for update
+A JSON object of order item IDs and an array containing the quantity.
```

Comment:
This text and the example are a mismatch with the reality - should be "A JSON object of order item IDs whose values are objects that define the item's new quantity." Then the example below should change the brackets to braces as in the curl example.